app/assets/stylesheets/gift_requests.scss

```diff
@@ -0,0 +1,27 @@
+.filter-box {
+  background-color: #eee;
+  margin: 5px;
+  border: 1px solid #ccc;
+  border-radius: 5px;
+  padding: 1em;
+
+  h4, h5 {
+    font-weight: bold;
```

Comment:
Minor CSS nit, but Bootstrap provides its own `<strong>` CSS rule that makes the font-weight bold already so you can use that instead, and nest it in the header tags, and just remove this CSS rule.